api/tests/test_regression/test_issue_482.py

```diff
@@ -0,0 +1,129 @@
+import json
+
+import requests
+
+from api.tests import APITestCase
+
+from ..util import load_content_as_json
+
+
+class Test482(APITestCase):
+    def setUp(self):
+        self._structure_data = {
+            "constraints": [
+                {
+                    "constraint_type": "PRIMARY KEY",
+                    "constraint_parameter": "id",
+                    "reference_table": None,
+                    "reference_column": None,
+                }
+            ],
+            "columns": [
+                {
+                    "name": "id",
+                    "data_type": "bigserial",
+                    "is_nullable": False,
+                    "character_maximum_length": None,
+                },
+                {
+                    "name": "name",
+                    "data_type": "character varying",
+                    "is_nullable": True,
+                    "character_maximum_length": 123,
+                },
+            ],
+        }
+
+        resp = self.__class__.client.put(
+            "/api/v0/schema/{schema}/tables/{table}/".format(
+                schema=self.test_schema, table=self.test_table
+            ),
+            data=json.dumps({"query": self._structure_data}),
+            HTTP_AUTHORIZATION="Token %s" % self.__class__.token,
+            content_type="application/json",
+        )
+
```

Comment:
A comment about what the assertion is about in plain language? (no need to be super long)

Reply:
Done in 592f8c7